tests/test_enrollment.py

```diff
@@ -51,6 +51,26 @@ def test_enroll_from_form(self):
         self.studentB['id'] = user.id
         
         self.enrollment_matches_info(user, self.studentB)
+
+    def test_unenroll(self):
+        self.setup_course()
+
+        user = User(name=self.studentA['name'], email=self.studentA['email'])
+        db.session.add(user)
+        db.session.commit()
+        self.studentA['id'] = user.id
+        
+        Enrollment.create(self.course.id, [self.studentA])
+        enrollment = Enrollment.query.filter_by(
+            course_id=self.course.id, 
+            user_id=user.id).one_or_none()
+        assert enrollment
+
+        Enrollment.unenroll(self.course.id, user.id)
+        enrollment = Enrollment.query.filter_by(
+            course_id=self.course.id, 
+            user_id=user.id).one_or_none()
+        assert not enrollment
```

Comment:
Also worth testing: Unenrolling a user.id that isn't enrolled and making sure that doesn't crash.